app/controllers/dishinstructions_controller.rb

```diff
@@ -0,0 +1,37 @@
+class DishinstructionsController < ApplicationController
```

Comment:
One thing about the name of this model. It already lives on a dish as a relationship so could this just be a `Instruction` model?